helm/values-prod.yaml

```diff
@@ -43,14 +43,14 @@ microservice-chart:
     allowPrivilegeEscalation: false
   resources:
     requests:
-      memory: "512Mi"
-      cpu: "0.25"
+      memory: "1536Mi"
+      cpu: "1.0"
```

Comment:
Is this value in CPU request in production correct? Seems too high, but I can be wrong.

Reply:
yes, since it is assumed it will have a similar load to [biz-events-to-datastore](https://github.com/pagopa/pagopa-biz-events-datastore/blob/868a2cb5fa50c01b76fb64f58268a542bd6e56ba/helm/values-prod.yaml#L47)

Reply:
@andrea-deri [it's equal to positive event](https://github.com/pagopa/pagopa-biz-events-datastore/blob/868a2cb5fa50c01b76fb64f58268a542bd6e56ba/helm/values-prod.yaml#L44) 
This configuration was the empirical result when it was put into prod in January.

I think the best thing here is to use KEDA on messages queue and not  cpu